.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -6,3 +6,8 @@ contact_links:
       Ask questions and discuss with other community members here.
       If you have a question like "How do I concatenate a list of datasets?" then
       please include a self-contained reproducible example if possible.
+  - name: Raster analysis usage question
+    url: https://github.com/corteva/rioxarray/discussions
+    about: |
+      If your question concerns analysis of raster datasets, for e.g.
+      reprojection, please ask it at the rioxarray repository.
```

Comment:
```suggestion
      If you are using the rioxarray extension (engine='rasterio'), or have questions about geospatial formats
      coordinate reprojection, etc., please use the rioxarray discussion forum.  
```

Reply:
not sure what's going on with the formatting, but thought this wording might be a bit clearer?

Reply:
Probably still want to keep the keyword "raster" in there though.